mtp_cashbook/apps/disbursements/views.py

```diff
@@ -787,6 +787,16 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(breadcrumbs_back=reverse('disbursements:process-overview'), **kwargs)
 
 
+class KioskInstructions(BaseView):
+    url_name = 'kiosk-instructions'
+
+    def get_context_data(self, **kwargs):
+        return super().get_context_data(
+            breadcrumbs_back=f"{reverse('disbursements:process-overview')}#section-fill-form",
+            **kwargs
```

Comment:
I think this errors if `kwargs` has `breadcrumbs_back`. It would be better to do something like
```
return super().get_context_data(
    **{
        **kwargs,
        breadcrumbs_back=f"{reverse('disbursements:process-overview')}#section-fill-form",
    },
}
```